src/main/java/org/apache/sling/feature/cpconverter/vltpkg/VaultPackageUtils.java

```diff
@@ -90,6 +90,7 @@ private VaultPackageUtils() {
 
     public static void setDependencies(@Nullable Set<Dependency> dependencies, @NotNull Properties properties) {
```

Comment:
maybe we should cleanup property `dependencies-locations` as well. In addition the Manifest may contain dependency information  as well (https://jackrabbit.apache.org/filevault/properties.html#Manifest_File), although this is not evaluated AFAIK